openshift/amunApiNamespace-template.yaml

```diff
@@ -10,14 +10,13 @@ metadata:
     template.openshift.io/documentation-url: https://github.com/Thoth-Station/
     template.openshift.io/long-description: This template defines resources needed to deploy Amun to OpenShift.
     template.openshift.io/provider-display-name: Red Hat, Inc.
-    thoth-station.ninja/template-version: 0.5.1
   labels:
     template: amun-core
     app: amun
 
 objects:
-  - kind: ServiceAccount
-    apiVersion: v1
+  - apiVersion: v1
```

Comment:
alphabetic sorting is nice, but bad if you use folding in your editor: you never know what is hidden because it simply says 'apiVersion: v1'

Reply:
Good point. Didn't think about that one.